src/main/java/io/confluent/idesidecar/restapi/messageviewer/data/SimpleConsumeMultiPartitionRequest.java

```diff
@@ -57,4 +61,12 @@ public Properties consumerConfigOverrides() {
     }
     return props;
   }
+
+  public String toJsonString() {
+    try {
+      return OBJECT_MAPPER.writeValueAsString(this);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
```

Comment:
The Quarkus framework would have rejected badly formatted JSON before we ever get here. I only had to wrap this an throw the `RuntimeException` so that it could be wrapped in a `Supplier` like `() -> <..>.toJsonString()`.